frontend/packages/kubevirt-plugin/src/utils/utils.ts

```diff
@@ -51,3 +52,20 @@ export const getVMLikeModelDetailPath = (
   namespace: string,
   name: string,
 ) => `${getVMLikeModelListPath(isCreateTemplate, namespace)}/${name}`;
+
+export const getVMWizardCreateLink = (
```

Comment:
can this live rather in `./url.ts` or `./endpoints.ts` or something like that

`utils/utils.ts` should only for the most random stuff we can't place anywhere else

Reply:
moved